server/core/mount.go

```diff
@@ -0,0 +1,31 @@
+package core
+
+type (
+	// Mount defines `mounts` db table.
+	Mount struct {
+		ID           uint       `gorm:"primary_key;auto_increment;not null" json:"id"`
+		Host         string     `gorm:"not null" json:"host"`
+		Docker       string     `gorm:"not null" sql:"type:text" json:"docker"`
```

Comment:
can you name this property `Container` instead of `Docker`? Also, please remove `sql:"type:text"`.